docusaurus-search-local/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@easyops-cn/docusaurus-search-local",
-  "version": "0.29.0",
+  "version": "0.30.0",
```

The version will be increased automatically, so we should leave as it is

```suggestion
  "version": "0.29.0",
```